vshard-scm-1.rockspec

```diff
@@ -10,7 +10,12 @@ description = {
     license  = 'BSD',
 }
 dependencies = {
-    'tarantool >= 1.9.0';
+    'lua >= 5.1';
```

Comment:
Why? Vshard obviously depends on Tarantool, not just on Lua.

Reply:
I wrote my reasoning already in https://github.com/tarantool/rocks/pull/42#issuecomment-482983290

Reply:
We need to check Tarantool version, because VShard does not work correctly on Tarantool < 1.9. And someday we will deprecate 1.9 support too as so on. If I can't do that via rockspec, then probably it can be done via CMakeLists.txt, but current implementation of find_packet(Tarantool) relies on tarantool/module.h header, which is not needed to VShard.

Reply:
We need to catch something like https://github.com/tarantool/stat/commit/cd70219fffc549fa6cb4e10f71e1b3cce4df8a22 and use tarantool version that is reported by `tarantool --version`.